RELEASE.md

```diff
@@ -1,3 +1,25 @@
+# Release 1.3.0
+
+## New features
+
+- Dockerfile optimisation (https://github.com/marqo-ai/marqo/pull/569/files). A pre-built Marqo base image results in reduced image layers and increased build speed.
+- New E5 models added to model registry (https://github.com/marqo-ai/marqo/pull/568). E5 V2 and Multilingual E5 models are now available for use. See all available models [here] (https://marqo.pages.dev/1.2.0/Models-Reference/dense_retrieval/).
+
+## Bug fixes and minor changes
+
+- Improved testing pipeline (https://github.com/marqo-ai/marqo/pull/582 & https://github.com/marqo-ai/marqo/pull/586). Tests now trigger on pull request updates. This results in safer and easier merges to mainline.
+- Major README overhaul (https://github.com/marqo-ai/marqo/pull/573). The README has been revamped with up-to-date examples and easier to follow instructions.
+- New security policy (https://github.com/marqo-ai/marqo/pull/574).
+- Updated requirements file. Now the `requirements.dev.txt` should be used to install requirements for development environments (https://github.com/marqo-ai/marqo/pull/569). Pins for `protobuf` & `onnx` have been removed while a pin for `anyio` has been added (https://github.com/marqo-ai/marqo/pull/581, & https://github.com/marqo-ai/marqo/pull/589).
+- General grammar and spelling fixes (https://github.com/marqo-ai/marqo/pull/577, https://github.com/marqo-ai/marqo/pull/578, https://github.com/marqo-ai/marqo/pull/587/files, & https://github.com/marqo-ai/marqo/pull/580)
+
+## Contributor shout-outs
+
+- A huge thank you to all our 3.5k stargazers!
+- Shoutout to @vladdoster for all the useful spelling and grammar edits!
+- Thanks everyone for keeping our [forum](https://community.marqo.ai/) alive and bustling. Don't hesitate to keep posting your insights, questions, and feedback!
```

Comment:
nit: `... alive and bustling` -> `... bustling`, it's not like our forum is on the edge of death!